Makefile

```diff
@@ -981,7 +988,7 @@ check-symbols: startup_files libc
 
 install: finish
 	mkdir -p "$(INSTALL_DIR)"
-	cp -r "$(SYSROOT)/lib" "$(SYSROOT)/share" "$(SYSROOT)/include" "$(INSTALL_DIR)"
+	cp -p -r "$(SYSROOT)/lib" "$(SYSROOT)/share" "$(SYSROOT)/include" "$(INSTALL_DIR)"
```

Comment:
How about using `cp -ar` everywhere?

Reply:
`-a` has subtle behavior differences across implementations (e.g. darwin cp does not preserve hard link but GNU coreutils does.) and it's not a part of POSIX spec unlike `-p` https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/utilities/cp.html
So I prefer `-p` unless we have clear needs for `-a`